spec/src/main/asciidoc/javase/bootstrap_se.asciidoc

```diff
@@ -101,6 +101,9 @@ If it is called and the container was already shutdown, it throws an `IllegalSta
 * `getBeanManager()` method returns the `BeanManager` (as defined in <<beanmanager>>) for the running container.
 If it is called and the container was already shutdown, it throws an `IllegalStateException`.
 
+SeContainer implements `javax.enterprise.inject.Instance` and therefore might be used to perform programmatic lookup as defined in <<dynamic_lookup>>.
+If no qualifier is specified in `Instance.select()` method the `@Default` qualifier is assumed.
```

Comment:
If no qualifier is passed to `SeContainer.select()` method the `@Default` qualifier is assumed.